test/test_cuda.py

```diff
@@ -3947,6 +3947,20 @@ def test_nvml_get_handler(self):
     def test_temperature(self):
         self.assertTrue(0 <= torch.cuda.temperature() <= 150)
 
+    @unittest.skipIf(TEST_PYNVML, "pynvml/amdsmi is not available")
+    def test_memory_usage_in_bytes(self):
+        """
+        Verify memory usage in bytes
+        """
+        torch.cuda.empty_cache()
+        a = torch.cuda.memory_usage_in_bytes()
+        num_bytes = 256 * 1024**2
+        _ = torch.zeros(num_bytes, dtype=torch.int8, device="cuda")
```

Comment:
Might want to switch this to `.empty()`, as if this is the first `zeros` call it could incur a kernel load which would increase the apparent device memory usage.

Reply:
updated to empty in the new version.